lldb/test/API/python_api/sbcoredumpoptions/TestSBCoreDumpOptions.py

```diff
@@ -0,0 +1,25 @@
+"""Test the SBCoreDumpOptions APIs."""
+
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+
+
+class SBCoreDumpOptionsAPICase(TestBase):
+    def test_plugin_name_assignment(self):
+        """Test"""
+        options = lldb.SBCoreDumpOptions()
+        error = options.SetPluginName(None)
+        self.assertTrue(error.Fail())
```

Comment:
This shouldn't fail, setting it to None clears the setting. So this test should probably set it to something valid first, like "minidump", verify you get that name back, then set it to None, verify you get None back.